lib/game.rb

```diff
@@ -0,0 +1,91 @@
+    @deck << @card5 = Card.new(:spade, '6', 6)
+    @deck << @card6 = Card.new(:spade, '7', 7)
+    @deck << @card7 = Card.new(:spade, '8', 8)
+    @deck << @card8 = Card.new(:spade, '9', 9)
+    @deck << @card9 = Card.new(:spade, '10', 10)
+    @deck << @card10 = Card.new(:spade, 'Jack', 11)
+    @deck << @card11 = Card.new(:spade, 'Queen', 12)
+    @deck << @card12 = Card.new(:spade, 'King', 13)
+    @deck << @card13 = Card.new(:spade, 'Ace', 14)
+    @deck << @card14 = Card.new(:diamond, '2', 2)
+    @deck << @card15 = Card.new(:diamond, '3', 3)
+    @deck << @card16 = Card.new(:diamond, '4', 4)
+    @deck << @card17 = Card.new(:diamond, '5', 5)
+    @deck << @card18 = Card.new(:diamond, '6', 6)
+    @deck << @card19 = Card.new(:diamond, '7', 7)
+    @deck << @card20 = Card.new(:diamond, '8', 8)
+    @deck << @card21 = Card.new(:diamond, '9', 9)
+    @deck << @card22 = Card.new(:diamond, '10', 10)
+    @deck << @card23 = Card.new(:diamond, 'Jack', 11)
+    @deck << @card24 = Card.new(:diamond, 'Queen', 12)
+    @deck << @card25 = Card.new(:diamond, 'King', 13)
+    @deck << @card26 = Card.new(:diamond, 'Ace', 14)
+    @deck << @card27 = Card.new(:heart, '2', 2)
+    @deck << @card28 = Card.new(:heart, '3', 3)
+    @deck << @card29 = Card.new(:heart, '4', 4)
+    @deck << @card30 = Card.new(:heart, '5', 5)
+    @deck << @card31 = Card.new(:heart, '6', 6)
+    @deck << @card32 = Card.new(:heart, '7', 7)
+    @deck << @card33 = Card.new(:heart, '8', 8)
+    @deck << @card34 = Card.new(:heart, '9', 9)
+    @deck << @card35 = Card.new(:heart, '10', 10)
+    @deck << @card36 = Card.new(:heart, 'Jack', 11)
+    @deck << @card37 = Card.new(:heart, 'Queen', 12)
+    @deck << @card38 = Card.new(:heart, 'King', 13)
+    @deck << @card39 = Card.new(:heart, 'Ace', 14)
+    @deck << @card40 = Card.new(:club, '2', 2)
+    @deck << @card41 = Card.new(:club, '3', 3)
+    @deck << @card42 = Card.new(:club, '4', 4)
+    @deck << @card43 = Card.new(:club, '5', 5)
+    @deck << @card44 = Card.new(:club, '6', 6)
+    @deck << @card45 = Card.new(:club, '7', 7)
+    @deck << @card46 = Card.new(:club, '8', 8)
+    @deck << @card47 = Card.new(:club, '9', 9)
+    @deck << @card48 = Card.new(:club, '10', 10)
+    @deck << @card49 = Card.new(:club, 'Jack', 11)
+    @deck << @card50 = Card.new(:club, 'Queen', 12)
+    @deck << @card51 = Card.new(:club, 'King', 13)
+    @deck << @card52 = Card.new(:club, 'Ace', 14)
+    @deck.shuffle
+  end
+
+  def start 
+    p "Welcome to War! (or Peace) This game will be played with 52 cards."
+    p "The players today are Megan and Aurora."
+    p "Type 'GO' to start the game!"
+    p '------------------------------------------------------------------'
+  end
+
+  start = gets.chomp
+
+
+  # I am struggling with how to execute actual game play. have reached
+  #out to my mentor for additional support in understanding how to 
+  #incorporate everything to get the actual game to run. 
+  # 
```

Comment:
Ok so now that we have hopefully got correct input from the user it is time to run the game...

```ruby
def start
  # I'm just hiding the previous code in this method for space, it is still here...

  turn_num = 1

  until @player1.has_lost? || @player2.has_lost? || turn_num == 1_000_000 # <== We want this loop to keep going until one of the player loses or we reach 1000000 turns
    turn = Turn.new(@player1, @player2) # <== For each iteration of the loop we want to create a new turn, passing in the same players except they will have less and less cards each time after the spoils have been awarded
    
    # Next we want to check the `turn.type` and print out the correct phrase by that type
    if turn.type == :basic
      p "Turn #{turn_num}: Whatever the phrase for a basic turn was..."
    elsif turn.type == :war
      p "Turn #{turn_num}: Whatever the phrase for a war turn was..."
    elsif turn.type == :mutually_assured_destruction
      p "Turn #{turn_num}: Whatever the phrase for a MAD turn was..."
    end

    # Afterwhich we will pile the cards, award the spoils of war cards to the winner, increase the turn count by 1 and begin again
    turn.pile_cards
    turn.award_spoils(turn.winner)
    turn_num += 1
  end

  # Once someone has lost, we need to figure out who that player was and print the appropriate output
  if @player1.has_lost?
    p "#{@player2.name} wins!"
  else
    p "#{@player1.name} wins!"
  end

  if turn_num == 1_000_000
    p "-----DRAW-----"
  end
end
```

Now I'm not sure if the code will work as I've written here exactly but this is just about the gist of it.